torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -664,21 +664,36 @@ def inner_fn(fn, args, *, use_trace_joint: bool):
 
         # Step 3: Unwrap any subclass outputs back into dense tensors
         unwrapped_outs = unwrap_tensor_subclasses(
-            wrapped_outs, is_joint_structure=use_trace_joint
+            wrapped_outs,
+            subclass_metas=None,
+            is_joint_structure=use_trace_joint,
+            is_runtime=False,
+            append_extra=append_extra,
         )
         return unwrapped_outs
 
     def joint_fn(primals, tangents):
-        return inner_fn(flat_fn_maybe_joint, (primals, tangents), use_trace_joint=True)
+        return inner_fn(
+            flat_fn_maybe_joint,
+            (primals, tangents),
+            use_trace_joint=True,
+            append_extra=False,
```

Comment:
I still can't think of a reason to avoid appending the extra size-y arguments - any updates on whether you were able to kill the flag?